src/applications/travel-pay/components/TravelPayClaimFilters.jsx

```diff
@@ -19,11 +22,30 @@ export default function TravelPayFilters(props) {
         header="Filter travel claims"
       >
         <div className="filter-your-results">
-          <TravelPayStatusFilters
```

Comment:
our idea with this was that we'd have `TravelPayStatusFilters` be a parent component to hold both the status checkboxes component and the date filter component. What do you think of making the date filter its own component, and then incorporating it into `TravelPayStatusFilters`?

Reply:
I second this - I didn't catch it in the status PR, but we should rename `<TravelPayStatusFilters />` to be more general: `<TravelPayFilterConterainer />` or something

Reply:
good call; keeps things more composable and clean, sets us up better if/when we add more to the filtering.